### DISCUSSION TITLE
Stale observation data

### DISCUSSION THREAD

**Discussion by paulnadler** [weather-gov/api#691] [Q&A]
All station data is stalled since Wednesday. `2023-12-27T14:53:00+00:00`. 

Is this related to the gridpoint outage?

Example request
https://api.weather.gov/stations/KCHA/observations/latest

**Comment by dwhitemv25** [comment]
Good eye, I just checked and my METARs stopped updating in one of my templates. I would think it's not directly related since the `/gridpoints` shutdown started 12/22, and the observations stopped 5 days later. 

I'm assuming the service that gets data from MADIS fell over and needs a kick, and nobody's noticed at NCO due to the holiday.

**Reply by JJKraw** [reply]
If METAR is sufficient as a work-around you can fetch from:

   https://tgftp.nws.noaa.gov/data/observations/metar/stations/{station}.TXT

E.g.
  https://tgftp.nws.noaa.gov/data/observations/metar/stations/KBOS.TXT

**Reply by jah1007** [reply]
Operational issues need to be reported to nco.ops@noaa.gov

**Reply by paulnadler** [reply]
I reported it to NCO before posting here, but was directed back referencing the gridpoint outage. That's why I asked here if they were related or not.

**Reply by jah1007** [reply]
It is not related.

**Reply by rr0ss0rr** [reply]
Just checked my app and it's also stalled at `https://api.weather.gov/stations/KROC/observations/latest` internally pointing to:
`"id": "https://api.weather.gov/stations/KROC/observations/2023-12-27T14:54:00+00:00",`

* I just added a sanity check to the app that if the station data is older than 12 hours then go with the Grid forecast.  My use basis is ok with this logic

**Comment by iegubkin** [comment]
Might be helpful: https://w1.weather.gov/xml/current_obs/KCHA.xml

**Reply by paulnadler** [reply]
That's the other API I use, but for the last several days it also hasn't been working. Generally 5-10 hours behind. Fingers cross it seems to be updating more frequently again today. I reported this to NCO several days ago. Not sure if they fixed it or it just started working again on its own.

**Comment by crsaulnier** [comment]
It seems that the ops team still doesn't realize this is a separate issue. I emailed this morning about it and they replied as if it was related to the gridpoints issue (that is now resolved).

**Comment by adamowsley** [comment]
It's alive!

"@id": "https://api.weather.gov/stations/KCHS/observations/2024-01-02T19:56:00+00:00",

**Reply by rr0ss0rr** [reply]
as well with KROC .. ;-)

**Comment by feelingwalnut** [comment]
KMPO current observations is finally back too!
